cmd/zdb/zdb.c

```diff
@@ -3023,6 +3027,117 @@ verify_dd_livelist(objset_t *os)
 	return (0);
 }
 
+static char *key_material = NULL;
```

Comment:
Double-checking, can't use libzfs_crypto's derive, as libzfs is not linked with zdb, only libzfs_core .. ?

Reply:
Partly that, though I could have just linked it in. The other reason is that in a recovery situation the keylocation might not be valid anymore, eg, you've moved the disks to another machine or sent them to a recovery shop.